client/state/experiments/hooks.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External Dependencies
+ */
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import { getVariationForUser, isLoading, getAnonId, nextRefresh } from './selectors';
+
+/**
+ * Returns true if the variations are loading for the current user
+ */
+export function useIsLoading(): boolean {
+	return useSelector( isLoading );
+}
+
+/**
+ * Returns the user's assigned variation for a given experiment
+ *
+ * @param experiment The name of the experiment
+ */
+export function useVariationForUser( experiment: string ): string | null {
+	return useSelector( ( state ) => getVariationForUser( state, experiment ) );
+}
+
+/**
+ * Gets the anon id for the user, if set
+ */
+export function useAnonId(): string | null {
+	return useSelector( getAnonId );
+}
+
+/**
+ * Get the time for the next variation refresh
+ */
+export function useNextRefresh(): number {
+	return useSelector( nextRefresh );
+}
```

Comment:
I wonder if we might use the idiomatic return for React? maybe something like:

```ts
// returns [ variation, isLoading, nextRefresh, anonId ]
function useExperiment( experiment: string ): [ string | null, boolean, number, string | null ];

// use like
const [ variation, isLoading, nextRefresh ] = useExperiment( experiment );
```

Though that might be some unnecessary state accesses. I'm happy either way.

Reply:
Good catch! I will add `useExperiment()` too!